### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by tituspijean**
## Problem
- Users may be at loss about how to describe their issue and how to report bugs.

## Solution
- To help us help them help us all (*à la* GLaDOS), I propose a simple issue template that guides the user. It first directs them between this repo, the SSOwat extension's and Flarum florum. Then important information and how to get them are listed.

## PR Status
- [x] Work finished.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- [x] **Code review** : frju365
- [x] **Approval (LGTM)** : frju365
- [ ] **Approval (LGTM)** :

### CODE REVIEW

**Comment by tituspijean** [comment]
I am quite open to discuss the usefulness of this addition. It could be a template for the other apps of the project.

If we go to add it, maybe we could add the `.github` folder to `.gitignore` ? No need to make the user download this.

**Review by frju365 — APPROVED** [review]
LGTM & Code Review ok. It's a good template

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   1. *Go to '...'*
+   2. *Click on '....'*
+   3. *Scroll down to '....'*
+   4. *See error*
+
+**Expected behavior**
+*A clear and concise description of what you expected to happen. You can remove this section if the command above is enough to understand your intent.*
+
+**Logs**
+*Perform the command again with `--debug | tee flarum.log` at the end. The log will be available in the `flarum.log` file. Remove any personal information and credentials, and copy and paste it here in a code block. If the log is long, you can use the [YunoPaste](https://paste.yunohost.org) service.*
```
Comment (tituspijean) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Actually `| tee` prevents the CLI to ask the user questions.

**Comment by frju365** [comment]
@tituspijean : Perhaps we can merge it ?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (frju365) [reply .github/ISSUE_TEMPLATE/bug_report.md]: ... yes. I don't know, but now I think when it fails the CLI ask you a command to paste the log, no ?

**Comment by tituspijean** [comment]
@frju365 no, with YNH last version now we can easily access the logs. We had a pending question about those, I am having a look about how to reformulate it.

**Comment by tituspijean** [comment]
There, @frju365, what do you think? I changed the last paragraph.

**Comment by frju365** [comment]
It's what I told you in the comment... so yes, I agree :P 

More seriously, yes it's completely ok. Like that we have no more problem.

**Review by frju365 — APPROVED** [review]

**Comment by tituspijean** [comment]
Oh sorry! I did not see your comment about it in the conversation. Let's merge!

**Pull request merged**